### DISCUSSION TITLE
how do I get the height of the status bar?

### DISCUSSION THREAD

**Discussion by qous** [NativeScript/NativeScript#10589] [General]
I need to get the height of the status bar from andoird, I use Native Script Vue (js), chat GPT suggests that I get it from a module that does not have this function. Is there any working suggestion on how to get the height of the bar status?

**Comment by vallemar** [comment]
Hi! for android you can do:

```ts
import { onMounted } from "nativescript-vue";
import { alert, Utils } from "@nativescript/core";

onMounted(() => {
  let resultInPixel = 0;
  const resourceId = Utils.android
    .getResources()
    .getIdentifier("status_bar_height", "dimen", "android");

  if (resourceId > 0) {
    resultInPixel = Utils.android
      .getResources()
      .getDimensionPixelSize(resourceId);
  }

  const resultDIP = Utils.layout.toDeviceIndependentPixels(resultInPixel);
  alert("result is " + resultInPixel + " " + resultDIP);
});
```

based on: https://stackoverflow.com/a/14213035/6015400

**Reply by qous** [reply]
hi, can u help me again ? https://github.com/orgs/NativeScript/discussions/10596
